Add Indonesian translation (#8)

hizkifw opened the pull request:
See #8

hizkifw committed:
Add some Indonesian translations

hizkifw committed:
Finish Indonesian translations

TiagoDanin commented on Baconit/Strings/id-ID/Resources.resw:
```diff
@@ -0,0 +1,889 @@
+    <value>Sepanjang Waktu</value>
+  </data>
+  <data name="CancelCT.Content" xml:space="preserve">
+    <value>BAtal</value>
```
`BAtal` is correct ?

hizkifw replied:
Ah, sorry, that one's a typo. It's supposed to be `Batal`.

hizkifw committed:
Fix capitalization

hizkifw committed:
Fix formatting and some links

hizkifw committed:
Merge pull request #1 from HizkiFW/readme-formatting-fix

Fix formatting and some links

hizkifw commented:
I've also fixed the broken formatting on the `README.md`.

Changes:
- Fixed broken headings
- Changed screenshots URL to this repo
- Fixed broken subreddit URLs
